### PR TITLE
Feat/1696

### PR DESCRIPTION
**Pull request by Kolezhanchik**
added empty list, fixed project=>product issue, fixed updating list after adding/deleting a user to one of aws roles list
image

### CODE REVIEW

File: app/(product)/public-cloud/(product)/aws-roles/[licencePlate]/[role]/page.tsx [hunk 1]
```diff
@@ -6,6 +6,47 @@ import TableBodyAWSRoles from '@/components/table/TableBodyAWSRoles';
 import { capitalizeFirstLetter } from '@/components/utils/capitalizeFirstLetter';
 import { useQuery } from '@tanstack/react-query';
 import { useSearchParams, useParams } from 'next/navigation';
+import UserAWSRolesTableTop from '@/components/table/TableTopUserAWSRoles';
+import AddUserModal from '@/components/modal/AddUser';
+import { useEffect, useState } from 'react';
+import DeleteUserModal from '@/components/modal/DeleteUser';
+import EmptyBody from '@/components/EmptyUsersList';
+
+async function addUser(userEmail: string, groupId: string): Promise<string | undefined> {
```
Comment (junminahn) [inline app/(product)/public-cloud/(product)/aws-roles/[licencePlate]/[role]/page.tsx:15]: Can we move all query services to `services` directory using `Axios` to separate `service` layer from the main page logic?
https://github.com/bcgov/platform-services-registry/blob/main/services/security-config.ts

and let's have `@tanstack/react-query` to handle errors.

File: app/(product)/public-cloud/(product)/aws-roles/[licencePlate]/[role]/page.tsx [hunk 2]
```diff
@@ -6,6 +6,47 @@ import TableBodyAWSRoles from '@/components/table/TableBodyAWSRoles';
...
+    if (response.ok) {
+      return response.statusText;
+    }
+    console.error('Failed to handle PUT request:', response.statusText);
+  } catch (error) {
+    console.error('Error during PUT request:', error);
+  }
+}
+
+async function deleteUser(userId: string, groupId: string): Promise<string | undefined> {
```
Comment (junminahn) [inline app/(product)/public-cloud/(product)/aws-roles/[licencePlate]/[role]/page.tsx:33]: Can we move all query services to `services` directory using `Axios` to separate `service` layer from the main page logic?
https://github.com/bcgov/platform-services-registry/blob/main/services/security-config.ts

and let's have `@tanstack/react-query` to handle errors.

File: app/(product)/public-cloud/(product)/aws-roles/[licencePlate]/[role]/page.tsx (hunk 1 above)
Comment (Kolezhanchik) [reply app/(product)/public-cloud/(product)/aws-roles/[licencePlate]/[role]/page.tsx]: all aws roles queries moved to services `@tanstack/react-query` used to handle errors

File: app/(product)/public-cloud/(product)/aws-roles/[licencePlate]/[role]/page.tsx (hunk 2 above)
Comment (Kolezhanchik) [reply app/(product)/public-cloud/(product)/aws-roles/[licencePlate]/[role]/page.tsx]: all aws roles queries moved to services `@tanstack/react-query` used to handle errors

**Review by junminahn — APPROVED** [review]

**Pull request merged**